Library/Homebrew/extend/ARGV.rb

```diff
@@ -120,6 +120,10 @@ def dry_run?
     include?("--dry-run") || switch?("n")
   end
 
+  def keep_files?
+    flag? "--keep-files"
+  end
```

Comment:
Yeah, the intention is for `install` to eventually make use of it, too; I just don't have an implementation for it yet.

Changed to `include?`.